scripts/cc-cli/src/test/integration-tests/wizard.test.ts

```diff
@@ -0,0 +1,29 @@
+import { newApi } from "../../api";
+import { parseAmountInternal } from "../../utils/parsing";
+import { getValidatorStatus } from "../../utils/validatorStatus";
+import { fundAccounts } from "./helpers";
+import execa from "execa";
+
+describe("integration test: validator wizard setup", () => {
+  test("new validator should appear as waiting after using the wizard setup", async () => {
+    // Fund stash and controller
+    const { stash, controller } = await fundAccounts(
+      parseAmountInternal("10000")
+    );
+    // Run wizard setup with 1k ctc ang to pair with node Bob
+    execa.commandSync(
+      `creditcoin-cli wizard --amount 1000 --url ws://localhost:9945`,
```

Comment:
I think it's good to use `ALICE_NODE_URL` and `BOB_NODE_URL` here too, to enforce good practices. 

Even better you can probably export those from helpers.ts